src/components/layout/GenericLayout/Header/index.tsx

```diff
@@ -56,17 +56,11 @@ const Logo = styled(Link)`
   }
 `
 
-interface Props {
-  menu?: React.ReactNode
-  tools?: React.ReactNode
-}
-
-export const Header: React.FC<Props> = ({ menu, tools }) => (
+export const Header: React.FC = ({ children }) => (
   <HeaderStyled>
     <Logo to="/" href="#">
```

Comment:
Shouldn't be the logo out too?

Reply:
I thought the logo was part of the generic header, no?

Reply:
I have some concerns regarding this... at what point is the reusablity/generic-icity(?) of the component(s) too much? Seeing as this is indeed `gp-ui`, emphasis on `gp` shouldn't some things be opinionated? In this case I'm referring to the `Logo` and maybe the placement of the wallet/menu stuff as it's always the same `Header` passed around `gp` apps.

Thoughts?

Reply:
I second that, and it's the point I raised last week in the chat https://gnosisinc.slack.com/archives/G01J17B9G6N/p1610737288016700

Reply:
My point is that this component is mixing too approaches. We had some discussion about this, I I think the conclusion was that for this container layout components, we try not to mix approaches. We either provide the content as children, or as parametrization of components.

I mean, it feels natural seeing the code you have to just pass the logo with the children, and it's more flexible. We can make small logo variations...

Anyways, I don't care that much about this, if we change one way or the other won't matter much. We can re-work later